org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/symbol/DocumentSymbolMapper.xtend

```diff
@@ -62,7 +62,7 @@ class DocumentSymbolMapper {
 	 * Provides {@link DocumentSymbol#detail detail} for a {@link DocumentSymbol document symbol}.
 	 * <p>
 	 * Always returns with an empty string by default.
-	 * 
```

Comment:
[nitpicky-remark]: That extra whitespace was there because that is the default `Xtend` formatting output in Eclipse. Can we switch back to the default one?

![screencast 2018-11-13 14-49-43](https://user-images.githubusercontent.com/1405703/48417768-debc8a00-e753-11e8-87d7-902b7d580425.gif)

Reply:
:+1: